config/nrfconnect/Kconfig

```diff
@@ -18,6 +18,30 @@ mainmenu "CHIP application"
 
 source "Kconfig.zephyr"
 
+config CHIP
+	bool "Enable CHIP"
+	default n
+	select CPLUSPLUS
+	select REBOOT
+	select ENTROPY_GENERATOR
+	imply NET_UDP
+	imply NET_IPV6
+	imply NET_CONFIG_NEED_IPV6
+	imply NET_SOCKETS
+	imply MULTIPROTOCOL
+	help
+	  Use the CHIP protocol.
+
+config NCS_SAMPLES_DEFAULTS
```

Comment:
I wouldn't put any content here which is only for samples. This file should be used even if one creates a final product. Move to overlay-sample-defaults.conf?

Reply:
Done